### DISCUSSION TITLE
How do I copy a position from frame to frame?

### DISCUSSION THREAD

**Discussion by hoshi411** [senshu/Sozi#619] [Q&A]
Somehow the position of one of my layers got destroyed across like 30 frames.
The position needs to be in the same place and size across all of the frames.  There is no way I can go to each frame and try to resize it to match size and position in every frame.

I tried using the "reset layer geometry" button but that did not reset it to original. It just made it super tiny.

I want to go to frame 1 and set the position and zoom and make that not change across all of the frames.
If I could copy and past that instance across to the other frames, that would work too.

**Comment by hoshi411** [comment]
The only way I solution I found was to use Inkscape.
Copy what is on that layer in cache.
Delete that layer.
Create a new layer.
Paste the cache.
Open in Sozi and add the new layer.

**Comment by tshoppa** [comment]
In case this happens again, here's an easier solution:

1. match the layer's view in the first frame 
2. select the layer in all frames (you can do it by clicking on the layer name on the left of the timeline)
3. Click on "link to previous frame" in the layer options on the right. This will copy the layer's first frame view to all frames.

(there will be a warning on the first frame because it cannot be linked to a previous frame. You can remove the "link" for that frame, but it does not matter anyway)

![image](https://user-images.githubusercontent.com/75733658/164401884-e687fbbf-a014-4058-b4d1-12baa2706bbc.png)

**Reply by hoshi411** [reply]
Thank you.  If I run into this problem again, I will try your answer but I do remember that even the first frame became corrupt. I may be wrong though.
